lib/express-recaptcha.js

```diff
@@ -78,7 +78,7 @@ Recaptcha.prototype.verify = function(req, cb){
     res.on('end', function() {
       var result = JSON.parse(body);
       var error = result['error-codes'] && result['error-codes'].length > 0 ? result['error-codes'][0] : 'invalid-input-response';
-      if (result.success) cb(null);
+      if (result.success) cb(null, result);
```

Comment:
I think that the data returned to the callback, must only contains hostname property.
like : { hostname : result.hostname }
no need to return the full object.